hackathon_site/event/serializers.py

```diff
@@ -10,7 +10,20 @@ class Meta:
         fields = ("id", "name")
 
 
+class TeamSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Team
+        fields = (
+            "id",
+            "team_code",
+            "created_at",
+            "updated_at",
+        )
+
+
 class ProfileSerializer(serializers.ModelSerializer):
+    team = TeamSerializer(read_only=True)
```

Comment:
I would actually recommend removing this, but leaving `team` in the fields list so it sticks around as an ID. I can't think of any case where we would need the full team to be serialized inside a profile endpoint.

Reply:
donezo